reference/curl/functions/curl-upkeep.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="utf-8"?>
+<refentry xml:id="function.curl_upkeep" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <refnamediv>
+  <refname>curl_upkeep</refname>
+  <refpurpose>Führt alle Prüfungen zur Aufrechterhaltung einer Verbindung durch</refpurpose>
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis>
+   <type>bool</type><methodname>curl_upkeep</methodname>
+   <methodparam><type>CurlHandle</type><parameter>handle</parameter></methodparam>
+  </methodsynopsis>
+  <para>
+   Verfügbar, wenn die Erweiterung mit libcurl &gt;= 7.62.0 kompiliert wurde.
+  </para>
+  <para>
+   Einige Protokolle haben Mechanismen um eine Verbindung aufrechtzuerhalten.
+   Diese Mechanismen senden in der Regel Daten an bestehende Verbindungen, um sie am Leben zu erhalten;
+   Dadurch kann verhindert werden, dass Verbindungen z. B. durch zu strenge Firewalls geschlossen werden.
```

Comment:
To avoid a line break I use an entity:
```suggestion
   Dadurch kann verhindert werden, dass Verbindungen &zb; durch zu strenge Firewalls geschlossen werden.
```